core/themes/basis/css/component/hero.css

```diff
@@ -26,6 +26,15 @@
   margin-top: -2rem;/* Negative top margin collapses spacing under header */
 }
 
+.update-1-30 .l-top .block-hero {
+  margin-top: 0;/* Negative top margin collapses spacing under header */
+}
+
+.update-1-30 .l-top .block-hero:first-child,
+.update-1-30 .l-top .block-system-messages:first-child {
```

Comment:
For the _system messages_ block, a top margin of `1rem` or `1.5rem` might be a better fit than `2rem`.